Packs/AWS-IAM/pack_metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "AWS - IAM",
+    "description": "Amazon Web Services Identity and Access Management (IAM)",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```

Comment:
```suggestion
    "serverMinVersion": "1.0.0",
```

Reply:
```suggestion
```

Reply:
missed it FSR... besides that the build passed